tests/smoke-tests/test-relay-indices.ts

```diff
@@ -107,6 +107,7 @@ describeSmokeSuite(
 
     testIt("C400", "should have matching indices for Staking.Bond", async function () {
       const callHex = context.relayApi.tx.staking
+        // @ts-ignore
```

Comment:
Why are those needed ?

Reply:
Get ts errors due to ApiAugment having mismatching function signatures. E.g.:
```
 staking: {
            /**
             * See [`Pallet::bond`].
             **/
            bond: AugmentedSubmittable<(value: Compact<u128> | AnyNumber | Uint8Array, payee: PalletStakingRewardDestination | {
                Staked: any;
            } | {
                Stash: any;
            } | {
                Controller: any;
            } | {
                Account: any;
            } | {
                None: any;
            } | string | Uint8Array) => SubmittableExtrinsic<ApiType>, [Compact<u128>, PalletStakingRewardDestination]>;
```
2 args, not 3. 
Will investigate further given that polkadot.js shows the fun as expected (maybe its an api-augment issue only).

Reply:
This is a result of function signatures changing in Polkadot RT9430, which won't be for a while.

Have changed this to @ts-expect-errors for now, and will remove when we need to update our fn signatures